Running ./school/Makefile under python3  results in error

When i execute `make html` in ./school/ I get the following error:

```
sphinx-build -v -b html -d build/doctrees   source build/html
Running Sphinx v1.4.1
loading translations [en]... done

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/Sphinx-1.4.1-py3.5.egg/sphinx/cmdline.py", line 243, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/usr/local/lib/python3.5/dist-packages/Sphinx-1.4.1-py3.5.egg/sphinx/application.py", line 191, in __init__
    self.config.check_types(self.warn)
  File "/usr/local/lib/python3.5/dist-packages/Sphinx-1.4.1-py3.5.egg/sphinx/config.py", line 314, in check_types
    default = default(self)  # could invoke l_()
  File "/usr/local/lib/python3.5/dist-packages/Sphinx-1.4.1-py3.5.egg/sphinx/config.py", line 241, in <lambda>
    make_filename(self.project).lower(),
  File "/usr/local/lib/python3.5/dist-packages/Sphinx-1.4.1-py3.5.egg/sphinx/util/osutil.py", line 157, in make_filename
    return no_fn_re.sub('', string) or 'sphinx'
TypeError: cannot use a string pattern on a bytes-like object

Exception occurred:
  File "/usr/local/lib/python3.5/dist-packages/Sphinx-1.4.1-py3.5.egg/sphinx/util/osutil.py", line 157, in make_filename
    return no_fn_re.sub('', string) or 'sphinx'
TypeError: cannot use a string pattern on a bytes-like object
The full traceback has been saved in /tmp/sphinx-err-uzbdu0nk.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:45: die Regel für Ziel „html“ scheiterte
make: *** [html] Fehler 1
```

--- a/school/source/conf.py
+++ b/school/source/conf.py
@@ -37,8 +37,8 @@ source_suffix = '.rst'
 master_doc = 'index'
 
 # General information about the project.
-project = b'iCSE for school'
-copyright = b'Marcin Kostur, iCSE-team'
+project = 'iCSE for school'
+copyright = 'Marcin Kostur, iCSE-team'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the

